test/js_test.go

```diff
@@ -2845,20 +2845,21 @@ func testJetStreamMirror_Source(t *testing.T, nodes ...*jsServer) {
 				err     error
 				seq     = uint64(i)
 				msg     *nats.RawStreamMsg
-				timeout = time.Now().Add(2 * time.Second)
+				timeout = time.Now().Add(5 * time.Second)
 			)
 
+		Retry:
 			for time.Now().Before(timeout) {
 				msg, err = js.GetMsg("s1", seq)
 				if err != nil {
 					time.Sleep(100 * time.Millisecond)
-					continue
+					continue Retry
 				}
 				msgs = append(msgs, msg)
 				continue GetNextMsg
 			}
 			if err != nil {
-				t.Fatalf("Unexpected error: %v", err)
+				t.Fatalf("Unexpected error fetching seq=%v: %v", seq, err)
```

Comment:
This is starting to flap a bit, waits 5 sec for a sequence to be present:
```
    js_test.go:2862: Unexpected error fetching seq=12: stream store EOF
```

Reply:
Next large PR should fix a bunch of those.